x-pack/plugins/uptime/public/components/overview/monitor_list/monitor_list.tsx

```diff
@@ -52,12 +54,30 @@ export const MonitorListComponent: ({
   setPageSize,
 }: Props) => any = ({ filters, monitorList: { list, error, loading }, pageSize, setPageSize }) => {
   const [drawerIds, updateDrawerIds] = useState<string[]>([]);
```

Comment:
It's unrelated to your changes but can we rename these?
```suggestion
  const [expandedDrawerIds, setExpandedDrawerIds] = useState<string[]>([]);
```